src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/Lightup/ProgrammaticSuppressionInfoWrapper.cs

```diff
@@ -12,21 +12,34 @@ namespace Microsoft.CodeAnalysis.Testing.Lightup
     {
         internal const string WrappedTypeName = "Microsoft.CodeAnalysis.Diagnostics.ProgrammaticSuppressionInfo";
         internal static readonly Type? WrappedType = typeof(Diagnostic).GetTypeInfo().Assembly.GetType(WrappedTypeName);
-        private static readonly Func<object, ImmutableHashSet<(string id, LocalizableString justification)>> s_suppressions;
+        private static readonly Func<object, object> s_suppressions;
 
         private readonly object _instance;
 
         static ProgrammaticSuppressionInfoWrapper()
         {
-            s_suppressions = LightupHelpers.CreatePropertyAccessor<object, ImmutableHashSet<(string id, LocalizableString justification)>>(WrappedType, nameof(Suppressions), ImmutableHashSet<(string id, LocalizableString justification)>.Empty);
+            s_suppressions = LightupHelpers.CreatePropertyAccessor<object, object>(WrappedType, nameof(Suppressions), ImmutableHashSet<(string id, LocalizableString justification)>.Empty);
         }
 
         private ProgrammaticSuppressionInfoWrapper(object instance)
         {
             _instance = instance;
         }
 
-        public ImmutableHashSet<(string id, LocalizableString justification)> Suppressions => s_suppressions(_instance);
+        public ImmutableHashSet<(string id, LocalizableString justification)> Suppressions
+        {
+            get
+            {
+                var suppressions = s_suppressions(_instance);
+                if (suppressions is ImmutableHashSet<(string id, LocalizableString justification)> set)
```

Comment:
>LocalizableString [](http://example.com/codeflow?start=65&length=17)

Just so I'm clear, there was an even earlier change that changed the signature that also would have broke this, right?

[https://github.com/dotnet/roslyn/commit/5287f2f0862a39cd07beaa07149aa3b43bae2f7a#diff-3dfff6578b6a50be75e51e45981adc7e43b19175986a528500cb0ba3d00e8553](https://github.com/dotnet/roslyn/commit/5287f2f0862a39cd07beaa07149aa3b43bae2f7a#diff-3dfff6578b6a50be75e51e45981adc7e43b19175986a528500cb0ba3d00e8553)

Reply:
That commit is the change which caused the break.